### PR TITLE
fixes issue #368 : camera switch button not working properl

### PR DESCRIPTION
**Pull request by amitiwary999**
Fixes issue #368 

Changes: When we replaced camera fragment by other fragment and then replacing that fragment by camera fragment, camera fragment start from initial i.e from Oncreateview and at oncreateview we are initializing camera by default . So I changed that so that camera will be updated accordingly. Similiar for status bar. When we click discard button onresume method of camera fragment get called so I added code there to hide status bar.

Screenshots for the change:

### CODE REVIEW

File: app/src/main/java/vn/mbm/phimp/me/Camera2.java [hunk 1]
```diff
@@ -322,7 +324,7 @@ else if (locationManager.getLastKnownLocation(LocationManager.NETWORK_PROVIDER)!
 			lon = -1;
 		}
 		try{
-			mCamera = Camera.open();}catch(Exception e){
+			mCamera = Camera.open(PhimpMe.camera_use);}catch(Exception e){
```
Comment (duchuyctlk) [inline app/src/main/java/vn/mbm/phimp/me/Camera2.java]: It would look nicer if the clause `catch` is put a line below.

**Comment by phimpme-danger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Message">
          1 Message
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:book:</td>
      <td data-sticky="false">:tada: Thanks for the PR @amitiwary999 we hope you have changed the variables in the upload_apk_forks.sh its for easy moderation.</td>
    </tr>
  </tbody>
</table>

:link: Maintainers: the apk to test can be found at https://github.com/amitiwary999/phimpme-android/blob/apk/app-debug.apk

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

File: app/src/main/java/vn/mbm/phimp/me/Camera2.java (hunk 1 above)
Comment (amitiwary999) [inline app/src/main/java/vn/mbm/phimp/me/Camera2.java]: that try catch block is not added by me.

Comment (Abhi2424shek) [inline app/src/main/java/vn/mbm/phimp/me/Camera2.java]: Please catch it though. It improves the code standard, won't it?

Comment (amitiwary999) [inline app/src/main/java/vn/mbm/phimp/me/Camera2.java]: @Abhi2424shek It is within a try catch block. @duchuyctlk wants me to put catch clause in the next line but it is not related to my issue so I left it.

Comment (Abhi2424shek) [inline app/src/main/java/vn/mbm/phimp/me/Camera2.java]: I know that, but you could always do that, Anyways thanks for this, merging
